cmd/clusterawsadm/api/bootstrap/v1beta1/defaults.go

```diff
@@ -43,8 +45,13 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 
 // SetDefaults_BootstrapUser is used by defaulter-gen.
 func SetDefaults_BootstrapUser(obj *BootstrapUser) { //nolint:golint,stylecheck
-	if obj != nil && obj.UserName == "" {
-		obj.UserName = DefaultBootstrapUserName
+	if obj != nil {
```

Comment:
shouldn't we add this change in v1beta2 api?

Reply:
I mean we would no need this in v1beta1 API

Reply:
Err, I see what you mean. Yeah, we do. But that reminds me... I'm not sure how clusterawsadm works. If it requires my to create a conversion rule or not. 🤔

Reply:
Probably no conversion needed, since I'm not altering the API.

Reply:
yeah I guess no conversion needed

Reply:
would you be able to add this to v1beta2?

Reply:
> would you be able to add this to v1beta2?

sorry, I'm not sure I follow. There is no `v1beta2` under the `clusterawsadm`.

Reply:
There is v1beta2 under `cmd/clusterawsadm/api/bootstrap`

Reply:
I must be going blind. :DDDDD Please halp.

This is what I see:

<img width="352" alt="Screenshot 2023-02-20 at 15 22 08" src="https://user-images.githubusercontent.com/182850/220132428-c3f89b3f-0b7b-4683-8ab7-fed41a68556e.png">


For the life of me I can't see v1beta2.

Reply:
Never mind, I have gone blind, we can close this comment 😅 sorry for confusing you. I might have confused with other folder.

Reply:
:D :D No worries. That can happen hahaha. Way tooooo many api folders. :D